bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="androiddebugbridge" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>AndroidDebugBridge Binding</name>
+	<description>This is the binding for connect to Android devices using the Android Debug Bridge protocol.</description>
+	<author>Miguel Álvarez</author>
+
+	<config-description>
+		<parameter name="discoveryPort" type="integer" required="true">
+			<label>Discovery Port</label>
+			<description>Port used on discovery to connect to the device through adb.</description>
+			<default>5555</default>
+		</parameter>
+		<parameter name="discoveryReachableMs" type="integer" required="true">
+			<label>Discovery Reachable ms</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with up to 25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions
```suggestion
			<label>Discovery Reachable</label>
```